src/message/__tests__/messageActionSheet-test.js

```diff
@@ -147,11 +162,43 @@ describe('constructHeaderActionButtons', () => {
 
     const mute = deepFreeze([]);
 
+    const ownUser = deepFreeze({
+      email: 'name@domain.com',
+      is_admin: false,
+    });
+
     const buttons = constructHeaderActionButtons({
-      backgroundData: { mute, subscriptions },
+      backgroundData: { mute, subscriptions, ownUser },
       message,
     });
 
     expect(buttons).toContain('muteStream');
   });
+
+  test('show delete topic option if current user is an admin', () => {
```

Comment:
A test for the case where the user is not an admin is also needed.

Reply:
Done